src/screens/Home/header.js

```diff
@@ -3,14 +3,21 @@ import React from 'react';
 import logo from 'images/logo_white.png';
 import userImage from 'images/user-placeholder.jpg';
 
-const Header = () => {
+const Header = ( { onClickUser }) => {
```

Comment:
@rafayet-monon  How about renaming this to `onAvatarClick` or `onUserImageClick`?